src/Util/CsrfTokenManager.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Util;
```

Comment:
I don't like the `util` name, please try to find a more specific name for this. 

Helper, util, manager, service, ... are components that have no specific purpose and will result in god objects.